modules/openapi-generator/src/test/java/org/openapitools/codegen/java/JavaModelEnumTest.java

```diff
@@ -172,6 +172,7 @@ public void testEnumTestSchema() {
         codegen.setOpenAPI(openAPI);
 
         Schema enumTest = openAPI.getComponents().getSchemas().get("Enum_Test");
+        Assert.assertNotNull(enumTest);
```

Comment:
Do our tests cover this new code?
This model: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml#L1422
does not contain `x-all-of-name`
Also doesn't this test require that the schema not be null?
Down below we test that the CodegenModel generated form the schema contains vars, which means that the schema had to not be null to contain those vars.